maintenance_stock/views/stock_move_line_views.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="stock_move_line_view_search" model="ir.ui.view">
+        <field name="model">stock.move.line</field>
+        <field name="inherit_id" ref="stock.stock_move_line_view_search" />
```

Comment:
Permissions is missing
![imagen](https://user-images.githubusercontent.com/28590170/93875949-40e26080-fcd6-11ea-8786-9673fd9cbf5b.png)

Reply:
I don't really know it this issue should be solved: if a user belogs to "Inventory / User" can access Product Moves in general... but not to every maintenance request, because of record rules. That access error is fired because user, who can access requests, it's not following this one.

Should it be added a record rule to prevent it, or accessing the Product Move is still correct, even if belongs to a request you don't follow to?

Reply:
If the permission is added on the view, the error shouldn't happen